dashboards-observability/public/components/trace_analytics/components/services/services.tsx

```diff
@@ -13,24 +13,43 @@ import { filtersToDsl } from '../common/helper_functions';
 import { SearchBar } from '../common/search_bar';
 import { ServicesTable } from './services_table';
 
-interface ServicesProps extends TraceAnalyticsComponentDeps {}
+interface ServicesProps extends TraceAnalyticsComponentDeps {
+  appId?: string;
+  appName?: string;
+  page: 'dashboard' | 'traces' | 'services' | 'app';
+}
 
 export function Services(props: ServicesProps) {
+  const { appId, appName, parentBreadcrumb, page } = props;
   const [tableItems, setTableItems] = useState([]);
   const [redirect, setRedirect] = useState(true);
   const [loading, setLoading] = useState(false);
 
-  useEffect(() => {
-    props.chrome.setBreadcrumbs([
-      props.parentBreadcrumb,
-      {
+  const breadCrumbs = page === 'app' ? 
+  [
+    {
+      text: 'Application analytics',
+      href: '#/application_analytics',
+    },
+    {
+      text: `${appName}`,
+      href: `#/application_analytics/${appId}`,
+    },
+  ] : [
+    {
         text: 'Trace analytics',
         href: '#/trace_analytics/home',
       },
       {
-        text: 'Services',
-        href: '#/trace_analytics/services',
+        text: 'Dashboards',
+        href: '#/trace_analytics/home',
```

Comment:
this shouldn't change?

Reply:
Oh yea, don't know why that's there. Good catch!